Game/BaseCharacter.gd

```diff
@@ -3,6 +3,9 @@ class_name BaseCharacter
 
 #Base class that has all the functions
 signal stat_changed
+signal pain_changed(amount)
+signal lust_changed(amount)
+signal stamina_changed(amount)
```

Comment:
make these provide the old and new values rather than just a delta and I can merge ^^

Reply:
I was actually going to do that at first before I decided to be minimalist for some reason :D
Done.